client/src/api/location-api.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+export default async function userLocation(latitude, longitude) {
+    const addressObj = `https://maps.googleapis.com/maps/api/geocode/json?latlng=${latitude},${longitude}&key=${process.env.REACT_APP_GOOGLE_API_KEY}`;
+    axios.get(addressObj).then((response) => {
+      const address = response.data.results[0].formatted_address;
```

Comment:
document why index 0, in case in the future we have to refactor this. It's not immediately obvious why the first one is picked